plugins/dashboards/frontend/public/templates/widgets/note/widget.html

```diff
@@ -1,16 +1,16 @@
 <div v-bind:class="[componentId, 'clyd-widget', 'dashboards-notes-widget']">
     <div class="bu-level notes-widget-button">
 		<div class="bu-level-left"></div>
-        <div class="bu-level-right">
+        <div class="bu-level-right" style="z-index: 1">
             <slot name="action"></slot>
         </div>
     </div>
 	<div class="clyd-widget__content">
-		<div style="width: 100%; height: 100%" class="bu-is-flex bu-is-flex-direction-column bu-is-justify-content-center">
-			<div class="bu-pl-2 bu-pr-2" :style="widgetStyling" v-html="widgetText"></div>
-			<p v-if="mylink" :style="linkStyling" class="bu-p-2">
-				<a class="bu-pt-4 bu-is-clickable color-dark-blue-100" target="_blank" :href="mylink.link" v-html="mylink.text">{{mylink.text}}</a>
-			</p>
-		</div>
+        <div style="width: 100%; height: 100%;" class="bu-is-flex bu-is-flex-direction-column bu-is-justify-content-center">
+			<vue-scroll :ops="scrollNote">
+			    <div class="bu-pl-2 bu-pr-2" :style="widgetStyling" v-html="widgetText"></div>
```

Comment:
we should use class instead of styling here.
and also in other places

Reply:
https://github.com/Countly/countly-server/pull/3581/commits/0d4a7a2afc9304ab6abcd79539ee1a30e2fe9700